include/translated-post.php

```diff
@@ -378,4 +373,29 @@ public function get_untranslated( $type, PLL_Language $untranslated_in, PLL_Lang
 
 		return $posts;
 	}
+
+	/**
+	 * Returns database-related informations that can be used in some of this class methods.
+	 * These are specific to the table containing the objects.
+	 *
+	 * @since 3.4.3
+	 * @see PLL_Translatable_Object::join_clause()
+	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
```

Comment:
Same as above.
```suggestion
	 * @see PLL_Translatable_Object::join_clause()
	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
	 *
	 * @since 3.4.3
```